hhs_oauth_server/settings/base.py

```diff
@@ -476,8 +475,7 @@
 
 MEDICARE_LOGIN_URI = env('DJANGO_MEDICARE_LOGIN_URI ',
                          'https://dev2.account.mymedicare.gov/?scope=openid%20profile&client_id=bluebutton')
-MEDICARE_REDIRECT_URI = env(
-    'DJANGO_MEDICARE_REDIRECT_URI', 'http://localhost:8000/mymedicare/sls-callback')
+MEDICARE_REDIRECT_URI = 'http://localhost:8000/mymedicare/sls-callback'
```

Comment:
why remove the option to have this set via the environment?

Reply:
Because it's not being set via an environment variable. All these options on how to set things are obfuscating what's going on, and it turns out that this is an application critical variable.

Reply:
I agree about the obfuscation, particularly across many settings files, but leaving this as an env with a sane default will make life allot easier for future dev's local development.